src/JsonSchema/SchemaFactory.php

```diff
@@ -200,18 +200,13 @@ private function buildPropertySchema(Schema $schema, string $definitionName, str
         }
 
         $propertySchema = new \ArrayObject($propertySchema + $valueSchema);
-        if (DocumentationNormalizer::OPENAPI_VERSION === $version) {
-            $schema->getDefinitions()[$definitionName]['properties'][$normalizedPropertyName] = $propertySchema;
-
-            return;
-        }
```

Comment:
Same block of code removed at https://github.com/api-platform/core/pull/3759/files (I'll need to rebase after merging this)

Reply:
So do you want me to keep it then?